docs/tutorials/tutorial-0-Linux.rst

```diff
@@ -0,0 +1,84 @@
+Linux Ubuntu 16.04 Tutorial 0 - Hello, world!
+===============================================
```

Comment:
You've still got some overflow on the underlining; and the title is still Ubuntu-specific.